site/content/contribute/more-info/server/style-guide.md

```diff
@@ -159,6 +159,50 @@ func OtherFunction() {
 }
 ```
 
+### Errors
+
+Always add proper context to errors.
+
+#### Include the user input for validation errors.
+
+For example:
+
+`return fmt.Errorf("invalid export type, must be one of: csv, actiance, globalrelay")` does not include what was the input value entered by the user. It needs to be included.
```

Comment:
Should we not only have a bad example here, but also the improved code?

Reply:
Done